File: server/src/main/java/com/example/server/model/EventItemDto.java
package com.example.server.model;

public class EventItemDto {
    public Long id;
    public String date;  // yyyy-MM-dd
    public String text;
    public String icon;
    public Long authorId;

    public static EventItemDto from(CalendarEvent e) {
        EventItemDto d = new EventItemDto();
        d.id = e.getId();
        d.date = e.getDate().toString();
        d.text = e.getText();
        d.icon = e.getIcon();
        d.authorId = e.getAuthorId();
        return d;
    }
}

File: client/app/src/main/java/com/example/us/ui/screen/calendar/EventItemDto.kt
package com.example.us.ui.screen.calendar

import kotlinx.serialization.Serializable

@Serializable
data class EventItemDto(
    val id: Long? = null,
    val date: String,   // "yyyy-MM-dd"
    val text: String,
    val icon: String = "🎀",
    val authorId: Long? = null   // опционально, кто создал
)
